### PR TITLE
List movies series

### PR DESCRIPTION
**Pull request by Sven-Lyco**
Added the first features as in Userstory 
[US-01] list popular movie and series #1

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,5 @@
+REACT_APP_API_BASE_SERIES_URL='https://api.themoviedb.org/3/tv'
```
Comment (Quaquaro) [inline .env:1]: You forgot the file in the .gitignore.

File: .gitignore [hunk 2]
```diff
@@ -12,6 +12,7 @@
 /build
 
 # misc
+.vscode
 .DS_Store
```
Comment (Quaquaro) [inline .gitignore:16]: .env is missing here.

File: src/App.js [hunk 3]
```diff
@@ -1,7 +1,69 @@
+import { useState, useEffect } from 'react';
+import styled from 'styled-components';
+
+import PosterList from './components/PosterList';
+import ScreenReaderOnly from './components/ScreenReaderOnly';
+import WatchaHeader from './assets/images/header/watcha.svg';
+
+const {
+  REACT_APP_API_BASE_SERIES_URL,
```
Comment (Quaquaro) [inline src/App.js:9]: Nice solution!

File: src/components/PosterList.js [hunk 4]
```diff
@@ -0,0 +1,46 @@
+import styled from 'styled-components';
+
+import Poster from './Poster';
+
+export default function PosterList({ list, listName }) {
+  return (
+    <>
+      <ListHeader>{listName}</ListHeader>
+      <StyledList>
```
Comment (Quaquaro) [inline src/components/PosterList.js:9]: Add role="list" here for accessibility

File: src/components/ScreenreaderOnly.js [hunk 5]
```diff
@@ -0,0 +1,13 @@
+import styled from 'styled-components';
+
+export default styled.span`
```
Comment (Quaquaro) [inline src/components/ScreenreaderOnly.js:3]: I would recommend that you put this in GlobalStyles.js and set the class for the elements in question.

**Review by Quaquaro — COMMENTED** [review]
I'm not sure if this is the case, but could it be that you haven't set your environment variables in Vercel yet. If so, take a look at the screenshot. In any case, very decent work. Keep it up
Vercel-Settings
!

File: .env (hunk 1 above)
Comment (Sven-Lyco) [reply .env:1]: Hi Christoph, thank you very much!
The error was that the naming of the file was wrong with the upper and lower case.

I added the `.env` into `.gitignore`

File: src/App.js (hunk 3 above)
Comment (Sven-Lyco) [reply src/App.js:9]: Thank you! 😄

File: src/components/PosterList.js (hunk 4 above)
Comment (Sven-Lyco) [reply src/components/PosterList.js:9]: I had this in mind earlier and then forgot to add it, thank you very much

File: .gitignore (hunk 2 above)
Comment (Sven-Lyco) [reply .gitignore:16]: I added the file, but VSCode does not gray it out and still pushes it up with it. Do I still need to manually delete the file on GitHub?

Comment (Sven-Lyco) [reply .gitignore:16]: Problem solved :)

File: src/App.js [hunk 6]
```diff
@@ -1,7 +1,69 @@
...
+} = process.env;
+
+const popularSeriesUrl = `${REACT_APP_API_BASE_SERIES_URL}/popular?api_key=${REACT_APP_API_KEY}&language=${REACT_APP_API_LANGUAGE}`;
+const popularMoviesUrl = `${REACT_APP_API_BASE_MOVIES_URL}/popular?api_key=${REACT_APP_API_KEY}&language=${REACT_APP_API_LANGUAGE}&region=DE`;
+
 export default function App() {
+  const [popularSeries, setPopularSeries] = useState([]);
+  const [popularMovies, setPopularMovies] = useState([]);
+
+  useEffect(() => {
```
[the remark this answers is not in the document]
Comment (Sven-Lyco) [reply src/App.js:22]: Thank you :) I have this in mind for my next user story. I am not so familiar with useFetch yet. But I'll try my best :)

**Review by Quaquaro — APPROVED** [review]
Nice job!

**Pull request merged**